release-team/role-handbooks/communications/README.md

```diff
@@ -43,11 +44,57 @@ The list has to be rotated/actively managed every cycle. Submit a PR to update [
+* **Scheduling press activities and the post-release webinar with the CNCF.** The Communications Coordinator works with the CNCF to schedule press release activities around the release, and to schedule the release webinar (typically scheduled 3-4 weeks post-release).
+
+### Release blog
+
+The Communications Coordinator along with the Comms shadows are responsible for authoring the official Kubernetes Release Blog. This blog is the official statement of release. The release blog is the primary vehicle by which the release team communicates the major themes, known issues, and other aspects of the release to the community. 
+
+Start the draft with the team around week 12, striking a balance between the release content being fairly well set and having enough to time author the blog and have it reviewed. Ahead of review, file a PR on k/website and cc the release lead and other stakeholders as reviewers.
+
+It's up to you and your team regarding how best to do this, typically it works well to assign sections to different team members (e.g. Major Themes, User Highlights, Ecosystem Updates, etc) and have the lead pull it all together and manage the PR and reviews.
+
+### Feature blogs
+
+Tracking, facilitating, and organizing the publication of the Feature Blog series is a major deliverable of the Comms team. Feature blogs are opt-in for SIGs, and authored by enhancement developers and others close to the features.
+
+The first feature blog typically goes out on release day alongside or shortly after the release blog, and then are published one-at-a-time, typically 2-3 a week. The Comms team establishes the publication schedule. Note that blog PRs in k/website are dated, and automation will publish future-dated entries. This enables a PR process decoupled from blog publication date.
+
+As feature blogs are opted in, assign them to shadows and yourself for tracking and facilitation. The responsibility is to ensure the blog authors have the resources and information they need, including editorial and tech reviews once ready.
+
+Work closely with the SIG Docs Blogs team (connecting on `#sig-docs-blogs` and by attending meetings), as they are typically available for editorial reviews. Share with them the feature blog schedule and updates throughout the cycle.
+
+For tech reviews, reach out to authors and the sponsoring SIG to organize at least one tech review per blog post.
+
+### Mid-cycle deprecations and removals blog
+
+This blog is optional and will vary from release to release. Work with the rest of the release team ahead of the Code Freeze date to determine if a mid-cycle blog focused on feature deprecations and removals is warranted. If so, facilitate its creation and publication.
```

Comment:
We could link to some previous examples, eg:
- https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/
- https://kubernetes.io/blog/2021/07/14/upcoming-changes-in-kubernetes-1-22/

Reply:
It looks like @jlbutler linked to the 1.22 deprecations doc further down the page.

Reply:
I moved the example up into this section, and used both of these.